cmake/OpenCensusDeps.cmake

```diff
@@ -23,12 +23,17 @@ fetchcontent_declare(abseil
                      GIT_REPOSITORY
                      https://github.com/abseil/abseil-cpp
                      GIT_TAG
-                     master)
+                     2c8421e1c6cef0da9e8a20b01c15256ec9ec116d)
```

Comment:
Is this fixed by #307? Can this go back to master now?

Reply:
Unfortunately, no - I rebased on master and tested this now

Reply:
Why does this have to be pinned? Is it to fix a build break? If so, please add a TODO.

Reply:
I changed it back to master and it builds locally. Is ABSL [only] broken on Windows?

Reply:
Yes, as I wrote, it's broken on my machine and I'm on Windows.

Reply:
Sorry, I didn't realize it was Windows specific.